src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/MergeBuildManifestsTests.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Tests
+{
+    [TestFixture]
+    public class MergeBuildManifestsTests
+    {
+        // TODO: Find whatever is making this not work concurrently.
+        [Test]
+        public void TwoCompatibleBuildData()
+        {
+            PushMetadataToBuildAssetRegistry pushMetadata = new PushMetadataToBuildAssetRegistry();
```

Comment:
Do similar initalization for pushMetadata object in this class as you do in MergeSigningInfo

Reply:
I had it that way but I changed it when I started seeing the state issue with the one test. Doesn't seem to be related though, so I'm putting it back today.

Reply:
If you implement the [Setup] method that I commented on below, and you debug through the tests, the SetUp method should run for each test, so there shouldn't be a saved state of that object.